prediction_market_agent_tooling/gtypes.py

```
@@ -18,6 +19,7 @@
 )
 from web3.types import Wei as Web3Wei
 
+from prediction_market_agent_tooling.gtypes import ChecksumAddress
```

Ruff (0.13.1)

Redefinition of unused `ChecksumAddress` from line 7

(F811)